stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,123 @@
+[id="proc-aap-gcp-backup-and-restore"]
+
+= Ansible Automation Platform Backup and Restore Process
+
+In order to backup and restore your AAP deployment, it is vital to have your existing AAP Admin Secret name and value recorded somewhere safe. In addition to this, it is also extremely important that regular manual backups of the Cloud SQL DB instance and filestore backups are taken to ensure a deployment can be restored as close as possible to its previous working state. The playbooks backup and restore provides backup and restore support for the AAP on GCP foundation deployment. 
+
+**Please not that the restore process will deploy a new deployment with the filestore and SQL DB instance being restored to the backup specified.**
+
+Follow the steps below to ensure a smooth backup and recovery process.
+
+== Backup Process
+
+The backup process involves taking backup of Cloud SQL DB instance and filestore instance at a given point of time.
+
+=== Run backup playbook 
+
+The backup playbook require an active AAP on GCP foundation deployment to be running. The instructions below guide how run backup playbook.
+
+. Gather the following input variables
+.. *gcp_service_account_credentials_json_path*: Service account crendential json file path for google cloud account.
+.. *gcp_deployment_name*: Deployment name which you want to backup.
+.. *gcp_compute_region*: GCP region where foundation deployment deployed.
+.. *gcp_compute_zone*: GCP zone where foundation deployment deployed.
++
+Create an extra_vars.yaml file with the above parameters
```

Comment:
This line should go before, around line 19 to explain what you do with the input variables.